docs/DeveloperGuide.md

```diff
@@ -271,6 +272,34 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 ---
 
+## **Appendix: Planned Enhancements**
+
+Team Size: 5
```

Comment:
Why did you put this?

Reply:
![image](https://github.com/user-attachments/assets/ce8568fa-fc35-434d-b7dd-1eb527aba76a)
As specified in the course website